src/docs/arch-rts.dox

```diff
@@ -1,113 +1,95 @@
-/*! @arch_page arch-rts Rollback to Stable
+/*! @arch_page arch-rts Rollback to Stable (RTS)
 
-Rollback to stable is an operation that retains only the modifications that
-are stable according to the stable timestamp and recovered checkpoint snapshot.
-The checkpoint transaction snapshot details are saved at the end of every
-checkpoint are recovered and used as a recovered checkpoint snapshot.
+RTS is an operation that retains only the modifications that are considered \a stable. A
+modification is considered \a unstable if it has a durable timestamp greater than the stable
+timestamp or its transaction id is not committed according to the recovery checkpoint snapshot.
+The recovery checkpoint snapshot corresponds to the checkpoint transaction snapshot details saved at
+the end of every checkpoint.
 
-@section rts-overview Overview of rollback to stable
+RTS works for all storage types except for LSM trees.
 
-Rollback to stable scans each and every table present in the database except
-@ref arch-metadata "Metadata" to remove the modifications from the table
-that are more than stable timestamp and recovered checkpoint snapshot.
+@section rts-overview Overview of RTS
 
-In the process of removing newer modifications from the table, all the in-memory
-updates are aborted and the on-disk version updates are replaced with an update
-from history store otherwise the data store version is removed.
+RTS scans tables in the database to remove any unstable modifications. A table is selected by RTS if
+it meets one of the following conditions:
 
-Rollback to stable is performed in three phases
-1. WiredTiger startup
-2. WiredTiger shutdown
-3. Application initiated
+1. It is modified.
+2. It has prepared updates.
+3. It has updates from transactions greater than the recovery checkpoint snapshot (this is
+applicable only during the restart phase)
+4. One of the checkpoint durable timestamps (start/stop) is greater than the rollback timestamp.
+5. There is no durable timestamp in any checkpoint.
```

Comment:
We don't mention any durable timestamp on the checkpoint [page](https://source.wiredtiger.com/develop/arch-checkpoint.html) of the arch guide. Should we add more details?

Reply:
Yes. RTS decides whether a table is to be processed for unstable updates or not based on the `durable timestamp` saved in the checkpoint metadata. The older versions of database files don't have this metadata string stored in the metadata. So we want to avoid processing these tables as part of the RTS. 

How about mentioning "tables with no aggregated time window in the checkpoint metadata"?

Reply:
Thank you. One more question as I am getting confused with `So we want to avoid processing these tables as part of the RTS`.
Do we want to process tables with no aggregated time window in the checkpoint metadata or we don't?

Reply:
We do not want to process the tables with no aggregated time window. The tables with no aggregate time window are created in versions older than MongoDB 4.4.

Reply:
Ok thanks for clarifying because this section is about the tables that are selected by RTS. I should move that entry to the next section when the tables are skipped by RTS.

Reply:
Fixed in c07425749a4cedc16e24ae311c8c97d4cef2d2c2